docs/tutorials/get-started/blink-an-led.md

```diff
@@ -124,27 +127,25 @@ Now that your circuit is wired, reconnect your Pi to power.
 Before proceeding, be sure that you have [connected your Pi to the Viam app](/get-started/installation/#install-viam-server).
 
 Now it's time to configure your machine's components.
-First, go to the [Viam app](https://app.viam.com/) on your web browser and navigate to your machine's **Config** tab.
+First, go to the [Viam app](https://app.viam.com/) and navigate to your new machine's **CONFIGURE** tab.
```

Comment:
```suggestion
Go to the [Viam app](https://app.viam.com/) and navigate to your new machine's **CONFIGURE** tab.
```
[opt] I know it's out-of-diff but I think "first" should come out since the proceeding paragraph says "before"... and we don't actually say "next" at any point